test/index.js

```diff
@@ -1,17 +1,16 @@
 var assert = require('assert');
-var babel = require('babel');
+var babel = require('babel-core');
 var chalk = require('chalk');
 var clear = require('clear');
 var diff = require('diff');
 var fs = require('fs');
 
-require('babel/register');
+require('babel-register');
 
 var pluginPath = require.resolve('../src');
 
 function runTest() {
 	var output = babel.transformFileSync(__dirname + '/fixtures/actual.js', {
-		optional: ['runtime'],
 		plugins: [pluginPath]

```

Comment:
I guess you need to add back runtime transformer & presets here (`.babelrc` works only for plugin code itself, not for APIs used in tests).